src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -125,8 +125,8 @@ type StorageMinerActorCode struct {
     PreCommitSector(rt Runtime, info sector.SectorPreCommitInfo)  // TODO: check with Magik on sizes
     ProveCommitSector(rt Runtime, info sector.SectorProveCommitInfo)
 
-    SubmitVerifiedSurprisePoSt(rt Runtime, onChainInfo sector.OnChainPoStVerifyInfo)
-    SubmitVerifiedElectionPoSt(rt Runtime, onChainInfo sector.OnChainPoStVerifyInfo)
+    SubmitVerifiedSurprisePoSt(rt Runtime)
+    SubmitVerifiedElectionPoSt(rt Runtime)
```

Comment:
Maybe `Submit` is not the best name here - nothing is being put up for any kind of inspection. `Handle` or `Process`?